.github/workflows/continuous-integration.yml

```diff
@@ -319,7 +319,7 @@ jobs:
           fetch-depth: 0
       - name: Security Audit
         timeout-minutes: 30
-        uses: ./.github/workflows/security-audit
+        uses: actions/vsp-release-tools-actions@v1
```

Comment:
we should used `platform-release-tools` since they're dropping `vsp` from everything